mappings/net/minecraft/nbt/deserialize/EmptyVisitor.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_6845 net/minecraft/nbt/deserialize/EmptyVisitor
```

Comment:
This is a skipping visitor that skips by default. It is not empty as an implementation can choose to not skip certain elements.